pkgs/development/python-modules/pyside6-qtads/default.nix

```
@@ -0,0 +1,75 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cmake,
+  cmake-build-extension,
+  pyside6,
+  setuptools,
+  setuptools-scm,
+  shiboken6,
+  wheel,
+  python,
+  qt6,
+}:
+
+buildPythonPackage rec {
+  pname = "pyside6-qtads";
+  version = "4.3.1.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mborgerson";
+    repo = "pyside6_qtads";
+    rev = "v${version}";
+    hash = "sha256-WSthRtK9IaRDDFEtGMUsQwylD+iGdsZM2vkXBjt8+mI=";
+    fetchSubmodules = true;
+  };
+
+  # bypass the broken parts of their bespoke python script cmake plugin
+  patches = [ ./find-nix-deps.patch ];
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail @shiboken6@ ${shiboken6} \
+      --replace-fail @pyside6@ ${pyside6}
+  '';
+
+  buildInputs = [
+    qt6.qtbase
+    qt6.qtquick3d
+    shiboken6
+    pyside6
+  ];
```

```suggestion
  buildInputs = [
    qt6.qtbase
    qt6.qtquick3d
  ];
```

they are already in dependencies